deploy/group_vars/all_db/main.yml

```diff
@@ -3,19 +3,19 @@
 
 backup_metadata_scripts:
   -
-    "CREATE TABLE public.pg_dist_partition AS SELECT * FROM pg_catalog.pg_dist_partition;"
+    "CREATE TABLE IF NOT EXISTS public.pg_dist_partition AS SELECT * FROM pg_catalog.pg_dist_partition;"
```

Comment:
@risa Should we just drop the tables first?  The next time the upgrade is performed, if the tables already exist with old data, then these commands would not update the data and so the backup would be bad, right?

Reply:
The temp tables are dropped later if the upgrade completes. I had to add this in because the script failed after it took backups but before they were dropped. When I tried to rerun the job, it would complain the tables already existed.

Reply:
Ah, OK as long as they are dropped we are good, thanks!